scripts/checkstyle.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+import re
+import sys
+import os
+import argparse
+from pathlib import Path
+from typing import List, Dict, Any # compat
+
```

Comment:
Can we get a comment section here about Usage, capturing the same stuff you have in the PR title for future users who find this file?

Reply:
Thanks